app/models/rejection.rb

```diff
@@ -9,16 +11,20 @@ class Rejection < ActiveRecord::Base
     'prisoner_non_association',
     'prisoner_released',
     'slot_unavailable',
-    'visitor_banned',
-    'visitor_not_on_list',
+    BANNED,
+    NOT_ON_THE_LIST,
     'duplicate_visit_request'
   ].freeze
 
   belongs_to :visit
 
-  validates :reason, inclusion: { in: REASONS }
   validate :validate_reasons
 
+  # TODO: Delete me when the column has dropped
+  def self.columns
```

Comment:
The review app script to populate dummy data is broken and is preventing the deploy of the review app for the PR